cmd/gateway/azure/gateway-azure.go

```diff
@@ -488,6 +488,11 @@ func (a *azureObjects) StorageInfo(ctx context.Context, _ bool) (si minio.Storag
 	return si
 }
 
+// GetOBDLock - Not relevant to Azure backend.
+func (a *azureObjects) GetOBDLock(ctx context.Context) (minio.RWLocker, error) {
+	return nil, minio.NotImplemented{}
+}
```

Comment:
GetOBDLock() can be defined just in gateway-unsupported.go and removed from other gateway specific files. All the gateway objects inherit from minio.GatewayUnsupported
```
func (a GatewayUnsupported)GetOBDLock(ctx context.Context) (minio.RWLocker, error) {
	return nil, minio.NotImplemented{}
}
```